subgraphs/uniswap-forks/protocols/apeswap/config/networks/bsc/bsc.ts

```diff
@@ -63,7 +63,7 @@ export class ApeswapBscConfigurations implements Configurations {
     return FeeSwitch.ON;
   }
   getRewardIntervalType(): string {
-    return RewardIntervalType.TIMESTAMP;
+    return RewardIntervalType.BLOCK;
```

Comment:
why are we changing this? I thought these ones have been QAed

Reply:
I wanted to review this with the protocol specialists. It was my bad for including this in this PR. When I was looking over the documents, it looked like the intervals needed to be flipped for BSC and Polygon. I was unsure though since it passed QA.

Reply:
I am following up on this.

Reply:
BANANA token is not even being priced on BSC and it looks like the pricing on Polygon is way off. The current price from the subgraph is 10 cents while the lowest price I see on coingecko is not less than 60 cents. If I understand how APR is calculated, I think this should lead to issues as well.

Reply:
Fixing in PR #964.